advanced_functionality/autogluon-sagemaker-pipeline/README.md

```diff
@@ -0,0 +1,110 @@
+# Autogluon Tabular with SageMaker Pipeline
```

Comment:
Branding nit: `Autogluon` -> `AutoGluon`

Should it be `...with SageMaker Pipelines` plural instead of `with SageMaker Pipeline`?

Reply:
done!